ReactAndroid/src/main/java/com/facebook/react/modules/image/ImageLoaderModule.java

```diff
@@ -78,6 +85,9 @@ public void getSize(
       return;
     }
 
+    // Ensure FrescoHelpers.initialize was called.
+    this.initialize();
```

Comment:
Is this even needed? The module initialize method should always get called before getSize can be called.